requirements.in

```diff
@@ -3,13 +3,12 @@ cachetools
 edlib
 fastapi==0.88.0
 hydra-core >= 1.2.0
-lightning == 1.9.5
+lightning>=2.5.1,<2.6
```

Comment:
I generally wouldn't upper bound unless there is a known problem with a package version because it can make resolution much harder